### ISSUE TITLE
No "Effort" and "Assigned To" visible on cards

### ISSUE THREAD

**Issue by nilskliesing-hon** [nick-myers/azure-devops-board-cards#13]
There are no values for "Effort" and "Assigned To" on the cards, even if the additional fields to include are selected.
![image](https://user-images.githubusercontent.com/26323226/61382604-65aa9f00-a8ad-11e9-8edb-ab6fc8d30591.png)

**Comment by nick-myers** [comment]
@nilskliesing-hon you are correct that the values are not included at the moment, I apologise.  I was part way through implementation when another issue arose, which I fixed, released but didn't take the options off the configuration section.  I'll try and finish the implementation soon.
